tests/Functional/run.php

```diff
@@ -16,7 +16,7 @@
 
 try {
     $app = new Application($config, $logger);
```

Comment:
Tady je ta nestandardní část - ta by měla pryč - hlavně taky načítání configu nahoře a tahání datadiru.

Reply:
Application zahodim hned jak pridam chybejci testy pro nove tridy v #94. Tady uz to neresme.